### PR TITLE
updated the color and format for balance in overview page

### PR DESCRIPTION
**Pull request by leelatanniru**
This PR fixed issue #245

### CODE REVIEW

File: src/renderer/components/DetailPanel/DetailPanel.scss [hunk 1]
```diff
@@ -17,6 +17,10 @@
       min-width: 150px;
       vertical-align: top;
     }
+    td.Balance {
+      color: var(--color-tertiary);
+      font-size: 14px;
+    }
```
Comment (angle943) [inline src/renderer/components/DetailPanel/DetailPanel.scss:23]: This breaks the rule: https://thenewboston.com/style-guide/css#component-scope

File: src/renderer/components/DetailPanel/index.tsx [hunk 2]
```diff
@@ -19,14 +20,30 @@ interface ComponentProps {
...
-            <td>{key}</td>
-            <td>{value}</td>
-          </tr>
-        ))}
+        {items.map(({key, value}) => {
+          if (key === 'Balance') {
+            return (
+              <tr key={key}>
+                <td>{key}</td>
+                <td className={key}>{getBalance(value)}</td>
```
Comment (angle943) [inline src/renderer/components/DetailPanel/index.tsx:36]: This breaks the rule: https://thenewboston.com/style-guide/css#sass-amp-bem

File: src/renderer/components/DetailPanel/index.tsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, {FC, ReactNode} from 'react';
 import clsx from 'clsx';
 
 import {getCustomClassNames} from '@renderer/utils/components';
+import {isNumber} from 'util';
```
Comment (angle943) [inline src/renderer/components/DetailPanel/index.tsx:5]: Screen Shot 2020-09-03 at 8 47 12 PM

**Review by angle943 — CHANGES_REQUESTED** [review]
left a few comments about the styleguide

**Comment by leelatanniru** [comment]
@angle943  Please consider only commit 1b55c29 (addressed review comments).

**Pull request closed without merging**